Website/src/rendering/src/services/BoxeverService.tsx

```diff
@@ -84,6 +84,8 @@ declare global {
       target: string;
       cookie_domain: string;
     };
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    mootrack: any;
```

Comment:
I think we can redeclare `global` and its `Window` interface in SendService with just `mootrack` in there. This would be cleaner than mixing mootrack and boxever declarations in the same file.